src/django_components/extension.py

```diff
@@ -707,6 +686,29 @@ def _init_app(self) -> None:
         resolver = get_resolver(urlconf)
         resolver._populate()
 
+        # Flush stored events
```

Comment:
Had to change the order, so the flushing of events happens after the URLResolver was updated